### ISSUE TITLE
Chinese chanracters are not displayed

### ISSUE THREAD

**Issue by kreghek** [Levrault/LE-dialogue-editor#42]
**Describe the bug**
I'm writing dialogue on chinese language. But chanracters are not displayed in the node.

![image](https://user-images.githubusercontent.com/2405499/171771445-75031a4c-b199-4c74-9695-a94df9c776b4.png)

But they are in a saved json

![image](https://user-images.githubusercontent.com/2405499/171771514-c3f272a2-7251-4665-a863-c994ffb2f6b4.png)

**To Reproduce**
1. Create dialogue node.
2. Type chinese or other non-latin character.

**Expected behavior**
Localization works fine :)

** Files**
```
[path]

resource="[path-to-project]/test"
OSX=""
Windows="[path-to-project]/test"
UWP=""
X11=""

[configuration]

has_portrait=false
has_name=true
dialogue_character_limit=0
choice_character_limit=0

[locale]

current="zh"
selected=[ "en", "zh" ]
custom=[  ]

[variables]

characters=[ {
"name": "hq",
"portraits": [  ],
"uuid": "5c54ce77-3a52-4435-8fe6-3f2bee5ad662"
}, {
"name": "ber",
"portraits": [  ],
"uuid": "00e2976f-1284-4888-990b-f67cd55dc179"
} ]
files=[ {
"name": "test.json",
"path": "[path-to-project]/test/test.json"
} ]

[cache]

last_opened_file={
"name": "test.json",
"path": "[path-to-project]/test/test.json"
}

[info]

version="v1.1.3"
```

```
{"5a25411d-063f-4f1e-ab03-63420b5e03bb":{"name":"hq","character_uuid":"5c54ce77-3a52-4435-8fe6-3f2bee5ad662","text":{"en":"","fr":"","zh":"要"},"parent":"bd06c18b-4797-43d6-a89a-bd387bfe1cd1","choices":[{"text":{"en":"o1","fr":"","ru":"","zh":""},"next":"337b838f-44a8-43df-831a-a80c470798ae","parent":"5a25411d-063f-4f1e-ab03-63420b5e03bb"},{"text":{"en":"o2","fr":"","ru":"","zh":""},"next":"0d90f192-0efc-4ce4-8081-2051aa3ddd47","parent":"5a25411d-063f-4f1e-ab03-63420b5e03bb"},{"text":{"en":"щщщфывфывщo3","fr":"","ru":"123фывфывфывфывфывфыв\nфыв","zh":""},"next":"bd06c18b-4797-43d6-a89a-bd387bfe1cd1","parent":"5a25411d-063f-4f1e-ab03-63420b5e03bb"}]},"337b838f-44a8-43df-831a-a80c470798ae":{"name":"ber","character_uuid":"00e2976f-1284-4888-990b-f67cd55dc179","text":{"en":"haku","fr":"","zh":""},"parent":"29dd51a0-81b5-4e5a-8466-1f1f7ef30518"},"0d90f192-0efc-4ce4-8081-2051aa3ddd47":{"name":"ber","character_uuid":"00e2976f-1284-4888-990b-f67cd55dc179","text":{"en":"fakuфыв","fr":"","zh":""},"parent":"6efbfcd9-76d4-410e-809b-d9c1e10efd17","choices":[{"text":{"en":"wo overpower","fr":"","zh":"","ru":""},"next":"80796358-52e2-4fa3-b291-18226aa8447f","parent":"0d90f192-0efc-4ce4-8081-2051aa3ddd47"},{"text":{"en":"I have overpower","fr":""},"next":"8d8e2e51-0aba-4e00-9799-d941ac2df03c","parent":"966200a9-8293-4bf8-9519-9eb26ae80cbf","uuid":"19dfe969-d97c-4b03-8159-34898af410a2"},{"text":{"en":"I have overpower","fr":"","zh":"","ru":""},"next":"8d8e2e51-0aba-4e00-9799-d941ac2df03c","parent":"966200a9-8293-4bf8-9519-9eb26ae80cbf","uuid":"19dfe969-d97c-4b03-8159-34898af410a2"},{"text":{"en":"I have overpower","fr":"","zh":"","ru":""},"next":"8d8e2e51-0aba-4e00-9799-d941ac2df03c","parent":"966200a9-8293-4bf8-9519-9eb26ae80cbf","uuid":"19dfe969-d97c-4b03-8159-34898af410a2"}],"conditions":[{"parent":"0d90f192-0efc-4ce4-8081-2051aa3ddd47","next":"19dfe969-d97c-4b03-8159-34898af410a2"," overpower":{"value":20,"operator":"greater","type":"int"}}]},"bd06c18b-4797-43d6-a89a-bd387bfe1cd1":{"name":"hq","character_uuid":"5c54ce77-3a52-4435-8fe6-3f2bee5ad662","text":{"en":"ok i give you overpower","fr":"","zh":""},"parent":"dd3781fa-56b2-49af-b2d1-977c78eb38da","signals":{"parent":"bd06c18b-4797-43d6-a89a-bd387bfe1cd1","overpower":{"Number":"100"}},"next":"5a25411d-063f-4f1e-ab03-63420b5e03bb"},"80796358-52e2-4fa3-b291-18226aa8447f":{"name":"hq","character_uuid":"5c54ce77-3a52-4435-8fe6-3f2bee5ad662","text":{"en":"end without","fr":"","zh":""},"parent":"4ae56c4d-6066-4684-a95f-dbc5381c53ed"},"8d8e2e51-0aba-4e00-9799-d941ac2df03c":{"name":"hq","character_uuid":"5c54ce77-3a52-4435-8fe6-3f2bee5ad662","text":{"en":"you have overpower","fr":"","zh":""},"parent":"19dfe969-d97c-4b03-8159-34898af410a2","signals":{"parent":"8d8e2e51-0aba-4e00-9799-d941ac2df03c"}},"root":{"next":"5a25411d-063f-4f1e-ab03-63420b5e03bb"},"__editor":{"root":{"uuid":"root","offset":[40,140]},"dialogues":[{"uuid":"5a25411d-063f-4f1e-ab03-63420b5e03bb","offset":[260,160],"parent":"root"},{"uuid":"337b838f-44a8-43df-831a-a80c470798ae","offset":[1180,160]},{"uuid":"0d90f192-0efc-4ce4-8081-2051aa3ddd47","offset":[1140,540]},{"uuid":"bd06c18b-4797-43d6-a89a-bd387bfe1cd1","offset":[520,1240]},{"uuid":"80796358-52e2-4fa3-b291-18226aa8447f","offset":[2340,460]},{"uuid":"8d8e2e51-0aba-4e00-9799-d941ac2df03c","offset":[2520,840]}],"conditions":[{"uuid":"966200a9-8293-4bf8-9519-9eb26ae80cbf","offset":[1660,840],"parent":"0d90f192-0efc-4ce4-8081-2051aa3ddd47","collapsed":false,"has_choice":true,"data":{"parent":"0d90f192-0efc-4ce4-8081-2051aa3ddd47","next":"19dfe969-d97c-4b03-8159-34898af410a2"," overpower":{"value":20,"operator":"greater","type":"int"}}}],"signals":[{"uuid":"2fc5c140-0698-421a-9610-cbf65b3889b4","offset":[540,1720],"parent":"bd06c18b-4797-43d6-a89a-bd387bfe1cd1","data":{"parent":"bd06c18b-4797-43d6-a89a-bd387bfe1cd1","overpower":{"Number":"100"}}},{"uuid":"e6c187fc-b68d-4fd0-b26b-f05b617a87aa","offset":[3020,920],"parent":"8d8e2e51-0aba-4e00-9799-d941ac2df03c","data":{"parent":"8d8e2e51-0aba-4e00-9799-d941ac2df03c"}}],"choices":[{"uuid":"29dd51a0-81b5-4e5a-8466-1f1f7ef30518","offset":[680,240],"parent":"5a25411d-063f-4f1e-ab03-63420b5e03bb"},{"uuid":"6efbfcd9-76d4-410e-809b-d9c1e10efd17","offset":[700,500],"parent":"5a25411d-063f-4f1e-ab03-63420b5e03bb"},{"uuid":"dd3781fa-56b2-49af-b2d1-977c78eb38da","offset":[660,760],"parent":"5a25411d-063f-4f1e-ab03-63420b5e03bb"},{"uuid":"4ae56c4d-6066-4684-a95f-dbc5381c53ed","offset":[1680,540],"parent":"0d90f192-0efc-4ce4-8081-2051aa3ddd47","data":{"text":{"en":"wo overpower","fr":"","zh":"","ru":""},"next":"80796358-52e2-4fa3-b291-18226aa8447f","parent":"0d90f192-0efc-4ce4-8081-2051aa3ddd47"}},{"uuid":"19dfe969-d97c-4b03-8159-34898af410a2","offset":[2080,880],"parent":"966200a9-8293-4bf8-9519-9eb26ae80cbf","data":{"text":{"en":"I have overpower","fr":"","zh":"","ru":""},"next":"8d8e2e51-0aba-4e00-9799-d941ac2df03c","parent":"966200a9-8293-4bf8-9519-9eb26ae80cbf","uuid":"19dfe969-d97c-4b03-8159-34898af410a2"}}]}}
```

**Screenshots**
See above

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 10

**Additional context**

**Comment by Levrault** [comment]
Hi, it seems related to the Fodor's native font that does not support Chinese characters. I saw that font that seems to support a lot of characters. I will try to create a pr in the next days http://www.unifoundry.com/unifont/index.html

**Comment by Levrault** [comment]
Hi, a new build has been pushed https://github.com/Levrault/LE-dialogue-editor/releases/tag/v1.1.4
Should also be avaible on itch.io

I used Noto_Sans_SC. Should now support simplified chinese. Since I'm have zero knowledge on that language, feel free to tell me if I should use a different font.

![Jun-03-2022 09-17-15](https://user-images.githubusercontent.com/1162446/171908372-23c1bccd-2cc0-45fc-a628-98bd224a76cc.gif)

I will close the issue when I got you confirmation. Thanks
